package.json

```diff
@@ -34,7 +34,7 @@
     "react-toastify": "^7.0.4",
     "recharts": "^2.0.9",
     "styled-components": "^5.3.0",
-    "use-wallet": "^0.9.0",
+    "use-wallet": "1.22.4",
```

Comment:
Where did `1.22.4` come from?

https://www.npmjs.com/package/use-wallet indicates latest is `0.9.0`

https://github.com/aragon/use-wallet/releases indicates latest non pre-release is `0.8.1`